app/views/handlers/__init__.py

```diff
@@ -38,3 +39,8 @@ def individual_response_url(
                 journey=journey,
             )
     return None
+
+
+def is_first_block_in_individual_response(location, schema):
```

Comment:
I think we should move this to the content handler since it only applies to that.

Reply:
Done!